tests/test_paper.py

```diff
@@ -0,0 +1,164 @@
+import trio
+import pytest
+import tractor
+import math
+import os
+from shutil import rmtree
+from piker.config import get_app_dir
+from piker.log import get_logger
+from piker.clearing._messages import ( 
+    Order                                    
+)
+from uuid import uuid4
+from typing import (
+    AsyncContextManager,
+    Literal,
+)
+from functools import partial
+from piker.pp import (
+    open_trade_ledger,
+    open_pps,
+)
+from piker.clearing import (
+    open_ems,
+)
+from piker.clearing._client import (
+    OrderBook,
+)
+from piker.clearing._messages import (
+    BrokerdPosition
+)
+from exceptiongroup import BaseExceptionGroup
+
+log = get_logger(__name__)
+
+@pytest.fixture(scope="session")
+def paper_cleanup():
+    yield
+    app_dir = get_app_dir('piker')
+    if os.path.isfile(app_dir):       
+        rmtree(app_dir)
+        assert not os.path.isfile(app_dir)
+
+def test_paper_trade(
+    open_test_pikerd: AsyncContextManager,
+    paper_cleanup: None
+):
+
+    cleared_price: float
+    test_exec_mode='live'
+    test_account = 'paper'
+    test_size = 1 
+    test_broker = 'kraken'
+    test_brokers = [test_broker]
+    test_symbol = 'xbtusdt' 
+    test_fqsn = f'{test_symbol}.{test_broker}'
```

Comment:
hmm maybe we should add a helper for this like we have for unpacking in https://github.com/pikers/piker/blob/master/piker/data/_source.py#L96 ?